### PR TITLE
[P1 tests] [Team 2] Add Punctuation Tokenizer Tests

### PR DESCRIPTION
**Pull request by FuyaoLi2017**
What's your team number?
Team 2

What is the functionality being tested?
Punctuation Tokenizer

Describe your tests briefly:
test1 tests if multiple newlines and tabs works.
test2 tests whether the tokenizer can deal with emojis.
test3 tests whether consecutive punctuation works.

Does each test case have comments/documentation?
Yes

### CODE REVIEW

File: src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+public class Team2PunctuationTokenizerTest {
+
+    /**
+     *  test1 tests whether multiple newlines, tabs,together with spaces works
+     */
+    @Test
+    public void test1() {
+        String text = "UCI: \n \n a public research university located in Irvine, \t \t California!";
+        List<String> expected = Arrays.asList("uci", "public", "research", "university", "located","irvine","california");
```
Comment (LaytonW) [inline src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java]: Colon (`':'`) is not specified as a delimiter punctuation. See: https://github.com/UCI-Chenli-teaching/spring19-cs221-project/blob/d3587226af776caf2751974264615e6057fb165b/src/main/java/edu/uci/ics/cs221/analysis/PunctuationTokenizer.java#L24
```suggestion
        List<String> expected = Arrays.asList("uci:", "public", "research", "university", "located","irvine","california");
```

Comment (LaytonW) [inline src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java]: Colon (`':'`) is not specified as a delimiter punctuation. See: https://github.com/UCI-Chenli-teaching/spring19-cs221-project/blob/d3587226af776caf2751974264615e6057fb165b/src/main/java/edu/uci/ics/cs221/analysis/PunctuationTokenizer.java#L24
```suggestion
        List<String> expected = Arrays.asList("uci:", "public", "research", "university", "located","irvine","california");
```

File: src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    /**
+     *  test2 tests whether the punctuation tokenizer can identify some emojis and save them as a token
+     *  test string: 🐴 is a very cute horse head!
+     */
+    @Test
+    public void test2() {
+        String text = "\uD83D\uDC34 is a very cute horse head!";
+        List<String> expected = Arrays.asList("\uD83D\uDC34", "very", "cute", "horse", "head");
```
Comment (LaytonW) [inline src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java]: The word "very" is listed as a stop word. See: https://github.com/UCI-Chenli-teaching/spring19-cs221-project/blob/d3587226af776caf2751974264615e6057fb165b/src/main/java/edu/uci/ics/cs221/analysis/StopWords.java#L134
```suggestion
        List<String> expected = Arrays.asList("\uD83D\uDC34", "cute", "horse", "head");
```

File: src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        assertEquals("check whether the tokenizer can deal with emojis" ,expected, tokenizer.tokenize(text));
+    }
+
+    /**
+     *  test3: this test tests whether the punctuation tokenizer can deal with consecutive punctuations
+     */
+    @Test
+    public void test3() {
+        String text = "UCI : \na, public research university located in Irvine,California!!!!";
+        List<String> expected = Arrays.asList("uci", "public", "research", "university", "located","irvine","california");
```
Comment (LaytonW) [inline src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java]: Colon (`':'`) is not specified as a delimiter punctuation. See: https://github.com/UCI-Chenli-teaching/spring19-cs221-project/blob/d3587226af776caf2751974264615e6057fb165b/src/main/java/edu/uci/ics/cs221/analysis/PunctuationTokenizer.java#L24
```suggestion
        List<String> expected = Arrays.asList("uci", ":", "public", "research", "university", "located","irvine","california");
```

File: src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java (hunk 1 above)
Comment (FuyaoLi2017) [reply src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java]: Hi, I have updated the mistakes, thank you!

**Comment by NinCors** [comment]
Looks good to me. I think those tests are ready for merging.

**Pull request merged**